### PR TITLE
update on index liveview

### PR DESCRIPTION
**Pull request by TheWestDevop**
I added events for getting, add, edit, update, and delete post in the homepage LiveView

### CODE REVIEW

**Comment by htdat148** [comment]
Hi @TheWestDevop , you can remove the `Mnesia` folder in your merge request since it is created when you start the application

**Comment by TheWestDevop** [comment]
Ok, currently far away from the laptop now

File: .tool-versions [hunk 1]
```diff
@@ -0,0 +1 @@
+elixir 1.13
```
Comment (pau-riosa) [inline .tool-versions:1]: are we also going to add otp and node version here?

**Review by pau-riosa — COMMENTED** [review]
Just added some comments

File: .tool-versions (hunk 1 above)
Comment (TheWestDevop) [reply .tool-versions:1]: No, that file was created because I was using asdf for elixir version control

**Pull request closed without merging**